docs/proposals/configmap.md

```diff
@@ -293,12 +296,12 @@ spec:
         - name: config-map-volume
           mountPath: /mnt/config-map
   volumes:
-  - name: config-map-volume
-    configMap:
-      files:
-        - path: "etc/redis.conf"
-          configMapName: redis-volume-config
-          key: redis.conf
+    - name: config-map-volume
+      configMap:
+        configMapName: redis-volume-config
```

Comment:
For env we chose a LocalObjectRef, and for volumes we chose a Name.  That smells a bit.  LocalObjectRef allows for extension into cross-namespace references, whcih seems plausible for large, complex applications.  A simple name would NOT allow that.  I spoke to Brian this morning and I'm coming around to cross-namespace stuff being viable.

Should we be converting to LocalObjectRef here?